persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageDAOImpl.kt

```diff
@@ -391,7 +390,7 @@ internal class MessageDAOImpl internal constructor(
     }
 
     override suspend fun resetAssetUploadStatus() = withContext(coroutineContext) {
-        queries.resetAssetUploadStatus()
+        assetStatusQueries.resetAssetTransferStatus()
     }
```

Comment:
question: right now `resetAssetTransferStatus` is exactly the same as `resetAssetUploadStatus` so shouldn't we just keep only the first one and remove `resetAssetUploadStatus` just like `resetAssetDownloadStatus`? 🤔

Reply:
Removed